### PR TITLE
Adds take control / collapsed search input.

### PR DESCRIPTION
**Pull request by aarondrabeck**

### CODE REVIEW

File: demo/titanium-search-input-demo.ts [hunk 1]
```diff
@@ -30,10 +30,27 @@ export default class TitaniumSearchInputDemo extends PolymerElement {
...
     <titanium-search-input placeholder="Search" value="{{searchTerm}}"></titanium-search-input>
+
+
+    <h3> Collapsed </h3>
+    <titanium-search-input placeholder="Search"></titanium-search-input>
+
+    <h3> Wide </h3>
+    <titanium-search-input wide placeholder="Search"></titanium-search-input>
+
+    <h3> Prevent Collaspse </h3>
```
Comment (antwonthegreat) [inline demo/titanium-search-input-demo.ts]: spelling

File: demo/titanium-search-input-demo.ts [hunk 2]
```diff
@@ -30,10 +30,27 @@ export default class TitaniumSearchInputDemo extends PolymerElement {
         --titanium-search-input-focus-color: slategray;
         --titanium-search-input-border-color: lightgray;
     }
+
+    titanium-search-input[wide] {
+        --titanium-search-input-witdh: 500px;
```
Comment (unfairSurprisery) [inline demo/titanium-search-input-demo.ts]: width?

File: src/titanium-search-input.ts [hunk 3]
```diff
@@ -41,48 +67,44 @@ export class TitaniumSearchInput extends LitElement {
   :host {
     display: flex;
     flex-direction: column;
+    -webkit-transition: width .250s; /* Safari */
+    transition: width .250s;
+    width: var(--titanium-search-input-witdh, 250px); ;
```
Comment (unfairSurprisery) [inline src/titanium-search-input.ts]: width?

File: demo/titanium-search-input-demo.ts (hunk 2 above)
Comment (aarondrabeck) [reply demo/titanium-search-input-demo.ts]: updated.

File: src/titanium-search-input.ts (hunk 3 above)
Comment (aarondrabeck) [reply src/titanium-search-input.ts]: updated.

**Pull request merged**